### PR TITLE
Enable auto-approval for external prod

### PR DESCRIPTION
**Pull request by dperaza4dustbit**
Implements: [ASC-541](https://issues.redhat.com//browse/ASC-541)

Also changing the default target from m01 to rh01. There is no need to manually move workspaces after this change is merged

### CODE REVIEW

**Comment by dperaza4dustbit** [comment]
Change to draft to avoid a merge before I cleanup existing pending approval user signups. Please review and approve if you agree with changes. Once we are ready to merge I will remove from draft as the last step, right after I do the cleanup

**Review by hugares — APPROVED** [review]

**Review by filariow — APPROVED** [review]
lgtm

**Comment by dperaza4dustbit** [comment]
All pending approval usersignups have been cleanup, letting this PR merge to enable the feature

**Pull request merged**
